plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/execution/IPrioritized.java

```diff
@@ -0,0 +1,27 @@
+/******************************************************************************* 
+ * Copyright (c) 2016 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/ 
+package org.jboss.reddeer.junit.execution;
+
+/**
+ * Interface for prioritizing objects.
+ * 
+ * @author mlabuda@redhat.com
+ * @since 1.2.0
+ */
+public interface IPrioritized {
+
+	/**
+	 * Gets object priority.
+	 * 
+	 * @return priority
+	 */
+	long getPriority();	
```

Comment:
Please extend the JavaDoc by the information that the higher priority the sooner execution (this is explained in PriorityComparator but users will look at this method)